src/python_testing/TC_FAN_3_5.py

```diff
@@ -21,207 +21,1090 @@
+
+                # LowestOff tests
+                TestStep(6, "[FC] TH tests Step command behavior of the LowestOff field.",
+                         "- Setup a Step command with: -- Direction=Decrease -- Wrap=False -- LowestOff=True."),
+                TestStep(7, "[FC] Initialize the PercentSetting attribute to 100.",
+                         "- Verify that the SpeedSetting attribute value is set to SpeedMax. - Verify that the FanMode attribute value is set to High."),
+                TestStep(8, "[FC] Subscribe to PercentSetting, SpeedSetting, and FanMode attributes.",
+                         "Listen for attribute report updates."),
+                TestStep(9, "[FC] TH sends Step commands iteratively.",
+                         "- Verify that the PercentSetting attribute value reaches 0. - When the PercentSetting attribute value reaches 0, send an additional Step command to veiryf that the PercentSetting attribute value stays at 0."),
+                TestStep(10, "[FC] Read the resulting attribute reports from each subscription after the Step commands.",
+                         "- Verify that the attribute report values from each subscription are in descending order. - If the number of PercentSetting reports is greater than the number of FanMode reports: -- Verify that all the expected FanMode values are present in the reports in accordance with the FanModeSequence attribute value. - If the number of PercentSetting reports is greater or equal than the number of SpeedSetting reports: -- Verify that all the expected SpeedSetting values are present in the reports in accordance with the SpeedMax attribute value. - Save the resulting descending attribute report values from each subscription as a baseline for comparison with the ascending values from the next steps."),
+
+                TestStep(11, "[FC] TH tests Step command behavior of the LowestOff field.",
+                         "- Setup a Step command with: -- Direction=Increase -- Wrap=False -- LowestOff=True."),
+                TestStep(12, "[FC] Initialize the PercentSetting attribute to 0.",
+                         "- Verify that the SpeedSetting attribute value is set to 0. - Verify that the FanMode attribute value is set to Off."),
+                TestStep(13, "[FC] Subscribe to PercentSetting, SpeedSetting, and FanMode attributes.",
+                         "Listen for attribute report updates."),
+                TestStep(14, "[FC] TH sends Step commands iteratively.",
+                         "- Verify that the PercentSetting attribute value reaches 100. - When the PercentSetting attribute value reaches 100, send an additional Step command to veiryf that the PercentSetting attribute value stays at 100."),
```

Comment:
I don't think reaching 100% is a requirement, is it? Can you verify that in the spec? LowestOff = True definitely means percent 0, but the highest step might be not 100%.